src/Kunstmaan/AdminBundle/Tests/Helper/AdminRouteHelperTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Kunstmaan\AdminBundle\Tests\Helper;
+
+use Kunstmaan\AdminBundle\Helper\AdminRouteHelper;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Kunstmaan\NodeBundle\Router\SlugRouter;
+
+class AdminRouteHelperTest extends \PHPUnit_Framework_TestCase
+{
+    protected static $ADMIN_KEY = "admin";
```

Comment:
Please test with a wider test data set. It would be nice if you use `@dataProvider` annotation with various cases. Not just with a default :)

Please test at least these cases:

When prefix is `admin`:
* domain.tld/admin
* domain.tld/en/admin
* domain.tld/en/some_path/admin

When prefix is `vip`:
* domain.tld/vip
* domain.tld/en/vip
* domain.tld/en/some_path/vip